.testing/_tc4/build_grid.py

```diff
@@ -0,0 +1,66 @@
+import netCDF4 as nc
+import numpy as np
+
+
+nx=14;ny=10 # grid size
+depth0=100. #uniform depth
+ds=0.01 # grid resolution at the equator in degrees
+Re=6.378e6 # Radius of earth
+
+topo_=np.zeros((ny,nx))+depth0
+f_topo=nc.Dataset('topog.nc','w',format='NETCDF3_CLASSIC')
+ny,nx=topo_.shape
+f_topo.createDimension('ny',ny)
+f_topo.createDimension('nx',nx)
+f_topo.createDimension('ntiles',1)
+f_topo.createVariable('depth','f8',('ny','nx'))
+f_topo.createVariable('h2','f8',('ny','nx'))
+f_topo.variables['depth'][:]=topo_
+f_topo.sync()
+f_topo.close()
+
+x_=np.arange(0,2*nx+1)*ds # units are degrees E
+y_=np.arange(0,2*ny+1)*ds # units are degrees N
+x,y=np.meshgrid(x_,y_)
+
+dx=np.zeros((2*ny+1,2*nx))
+dy=np.zeros((2*ny,2*nx+1))
+rad_deg=np.pi/180.
+dx[:]=rad_deg*Re*(x[:,1:]-x[:,0:-1])*np.cos(rad_deg*y[:,1:])
```

Comment:
This is the same short cut that led to the wrong areas on the OM4 grids. Please do the centered numerics even if it shouldn't be necessary on a spherical grid - someone will copy it.